src/pages/markup.html

```diff
@@ -0,0 +1,272 @@
+{% extends '_layout.html' %}
+
+{% block title %} {{ super() }} | Markup {% endblock %}
+
+{% block content %}
+
+<img src="http://placehold.it/1000x250" height="250">
+
+<h1>Hello</h1>
```

Comment:
Yeah, I'd like to expand that page to show the styles. Or have a singular page to show all the styles. But I like that this one is much more complete.